build_tools/github_actions/test_executable_scripts/test_rccl.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import os
+import shlex
+import subprocess
+from pathlib import Path
+import pytest
+
+THEROCK_BIN_DIR = os.getenv("THEROCK_BIN_DIR")
+SCRIPT_DIR = Path(__file__).resolve().parent
+THEROCK_DIR = SCRIPT_DIR.parent.parent.parent
+logging.basicConfig(level=logging.INFO)
+
+
+class TestRCCL:
+    def test_rccl_unittests(self):
+        # Executing rccl gtest from rccl repo
+        cmd = [f"{THEROCK_BIN_DIR}/rccl-UnitTests"]
+        logging.info(f"++ Exec [{THEROCK_DIR}]$ {shlex.join(cmd)}")
+        result = subprocess.run(
+            cmd,
+            cwd=THEROCK_DIR,
+            check=False,
+        )
```

Comment:
These should log their output so developers can see which tests are running and what they are doing.

Before: https://github.com/ROCm/TheRock/actions/runs/16306004018/job/46061125576
After: https://github.com/ROCm/TheRock/actions/runs/16327496798/job/46121451422